--- origsrc/mkrf-0.2.3/lib/mkrf/availability.rb	2008-05-22 07:50:19.343750000 -0500
+++ src/mkrf-0.2.3/lib/mkrf/availability.rb	2008-05-22 08:07:27.921875000 -0500
@@ -81,7 +81,7 @@
     def has_library?(library, function = "main", *paths)
       logger.info "Checking for library: #{library}"
       return true if library_already_loaded?(library)
-      return true if RUBY_PLATFORM =~ /mswin/ # TODO: find a way on windows
+      return true if RUBY_PLATFORM =~ /mswin|cygwin/ # TODO: find a way on windows
       # Should this be only found_library? or a specialized version with
       # path searching?
       found_library?(library, function)
